test/room_test.rb

```diff
@@ -3,22 +3,32 @@
 require './lib/room'
 
 class RoomTest < Minitest::Test
+  def setup
+    @room1 = Room.new(:bedroom, 10, '13')
+    @room2 = Room.new(:living_room, 15, '12')
+  end
+
   def test_it_exists
-    room = Room.new(:bedroom, 10, '13')
-    assert_instance_of Room, room
+    assert_instance_of Room, @room1
   end
 
   def test_it_has_a_category
-    skip
-    room = Room.new(:bedroom, 10, '13')
-    assert_equal :bedroom, room.category
+    assert_equal :bedroom, @room1.category
   end
 
   def test_it_can_get_area
-    skip
-    room1 = Room.new(:bedroom, 10, '13')
-    room2 = Room.new(:living_room, 15, '12')
-    assert_equal 130, room1.area
-    assert_equal 180, room2.area
+    assert_equal 130, @room1.area
+    assert_equal 180, @room2.area
+  end
+
+  def test_it_starts_painted_and_can_be_painted
+    refute @room1.is_painted?
+    refute @room2.is_painted?
+
+    @room1.paint
+    @room2.paint
+
+    assert @room1.is_painted?
+    assert @room2.is_painted?
```

Comment:
👍 

Since these rooms are fundamentally the same, you probably only need to paint one room to feel pretty confident that that method works.